packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -269,41 +269,22 @@ class RenderAndroidView extends PlatformViewRenderBox {
   }
 }
 
-/// A render object for an iOS UIKit UIView.
-///
-/// [RenderUiKitView] is responsible for sizing and displaying an iOS
-/// [UIView](https://developer.apple.com/documentation/uikit/uiview).
-///
-/// UIViews are added as sub views of the FlutterView and are composited by Quartz.
-///
-/// {@macro flutter.rendering.RenderAndroidView.layout}
-///
-/// {@macro flutter.rendering.RenderAndroidView.gestures}
-///
-/// See also:
-///
-///  * [UiKitView] which is a widget that is used to show a UIView.
-///  * [PlatformViewsService] which is a service for controlling platform views.
-class RenderUiKitView extends RenderBox {
-  /// Creates a render object for an iOS UIView.
-  ///
-  /// The `viewId`, `hitTestBehavior`, and `gestureRecognizers` parameters must not be null.
-  RenderUiKitView({
-    required UiKitViewController viewController,
+/// Superclass for common functionality of platform view RenderBoxes.
+/// Provides the basic rendering logic for iOS and macOS platformviews.
+/// Subclasses shall override handleEvent in order to execute custom event logic.
+/// T represents the class of the view controller for the corresponding widget.
```

Comment:
Format nit: Extract out a summary sentence and break it into a separate paragraph. https://dart.dev/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph

I don't think we ever explicitly say to avoid using "superclass for" but I wonder if we can just shorten this to:

```dart
/// Common render-layer functionality for iOS and macOS platform views.
///
/// Further details in follow-on paragraph here.
```